### ISSUE TITLE
Convert to HCL2 generates invalid string concat expressions

### ISSUE THREAD

**Issue by jdillon** [VladRassokhin/intellij-hcl#212]
idea version: IntelliJ IDEA 2019.1.3 (Ultimate Edition)
intellij-hcl plugin version: 0.7.1
terraform version: Terraform v0.12.0

 * * *

When using convert to HCL2 expression with 0.11 syntax like:

```hcl
resource "aws_kms_alias" "main" {
  name = "alias/${var.alias}"
  target_key_id = aws_kms_key.main.arn
}
```

The plugin transforms this to:

```hcl
resource "aws_kms_alias" "main" {
  name = "alias/" + var.alias
  target_key_id = aws_kms_key.main.arn
}
```

Which causes terraform to complain:

```
Error: Invalid operand

  on ../../modules/kms-key/main.tf line 9, in resource "aws_kms_alias" "main":
   9:   name = "alias/" + var.alias

Unsuitable value for left operand: a number is required.
```

I also don't think there should even be suggested refactoring for string template usage like this.  While it could be transformed to a format() expression it seems like thats a bit heavy handed for simple string generation like this.

**Comment by VladRassokhin** [comment]
Thank you for pointing to that problem.

I'm actually embarassed that language does not support '+' as concatenation.

For now will disable expanding complex strings. But that feature was so promising 😞

**Comment by VladRassokhin** [comment]
For now you can disable 'Convert to HCL2' intention in settings or in 'shift-shift' menu

**Comment by VladRassokhin** [comment]
Fixed, will be available in 0.7.2

**Comment by jdillon** [comment]
BTW you may want to try and see if its just a local/variable or function ref, and then provide the hint to convert for those?.  at least 75% of strings I had, were for local/variable or functions calls and I made use of the convert to hcl2 feature for those.  But still a few more were better left as a string format expression.
